src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -137,6 +139,8 @@ static void print_help_messages() {
                               "the following order: number of iterations for pre-allocation(int), max size of single iteration in bytes(int), "
                               "max per-dim allowed diff(int), unconditional buffers preallocation ratio(float). For example for disabling memory"
                               "preallocation at all, you can use OV_GPU_MemPreallocationOptions='0 0 0 1.0'");
+    message_list.emplace_back("OV_GPU_LoadDumpRawBinary", "Specified inputs layers loading dumped raw binary before the execution of layer."
+                               " Separated by space");
```

Comment:
difficult to understand the usage. Could you add an example?

Reply:
Could you add an example? I think example would be more intuitive because it is unclear whether primitive type is included or not in the name.